hudi-flink/src/main/java/org/apache/hudi/sink/CleanFunction.java

```diff
@@ -90,4 +100,28 @@ public void snapshotState(FunctionSnapshotContext context) throws Exception {
   public void initializeState(FunctionInitializationContext context) throws Exception {
     // no operation
   }
+
+  private void removeCorruptedCleanAction() {
+    HoodieTableMetaClient client = writeClient.getHoodieTable().getMetaClient();
+    HoodieTimeline cleanerTimeline = client.getActiveTimeline().getCleanerTimeline();
+
```

Comment:
It is not that useful to only execute once in the `#open` method.

Reply:
Yes, execute this after every clean action is better.